etc/frontend/di.xml

```diff
@@ -0,0 +1,16 @@
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <type name="Magento\Customer\CustomerData\SectionPoolInterface">
+        <arguments>
+            <argument name="sectionSourceMap" xsi:type="array">
+                <item name="metrilo-private-data" xsi:type="string">Metrilo\Analytics\CustomerData\PrivateData</item>
```

Comment:
What does private data mean?

Reply:
What is customer data, either?

Reply:
private data = customer specific data per request
customer data is the suggested best practice namespace for module section updates functionality

Reply:
suggested where?

Reply:
https://devdocs.magento.com/guides/v2.3/extension-dev-guide/cache/page-caching/private-content.html